### PR TITLE
Completed and homogenized German translation

### PR DESCRIPTION
**Pull request by MarcelWaldvogel**
Completed German translation and tried to make texts homogeneous, informative, and compact.

### CODE REVIEW

**Comment by bures** [comment]
Thanks.

**Comment by bures** [comment]
Thanks.

File: locales/de-DE/common.json [hunk 1]
```diff
@@ -133,47 +133,47 @@
   "subscribers": "Abonnenten",
   "description": "Beschreibung",
   "namespace": "Namensraum",
-  "namespace_plural": "Namenräume",
+  "namespace_plural": "Namesnräume",
```
Comment (fthobe) [inline locales/de-DE/common.json:136]: @MarcelWaldvogel @bures Typo, correct spelling would be "Namensräume", but I am honestly not sure that a localized substitute for namespace would be appropriate as Namespace is common enough in German IT language.

**Review by fthobe — COMMENTED** [review]
Hey guys, I would suggest that I sit down this weekend and work over the translations. It's horrid.

**Comment by bures** [comment]
I would leave this to Germans to sort it out. :-) I'll merge what you give me.

**Pull request merged**